packages/suite/src/components/suite/NavigationBar/components/NavigationActions/components/ActionItem.tsx

```diff
@@ -99,58 +113,60 @@ interface IconComponentProps extends CommonProps {
     iconComponent?: never;
 }
 
-type Props = CustomIconComponentProps | IconComponentProps;
+type ActionItemProps = CustomIconComponentProps | IconComponentProps;
 
 // Reason to use forwardRef: We want the user to be able to close Notifications dropdown by clicking somewhere else.
 // In order to achieve that behavior, we need to pass reference to ActionItem
-export const ActionItem = React.forwardRef((props: Props, ref: React.Ref<HTMLDivElement>) => {
-    const theme = useTheme();
-    const iconComponent = props.icon ? (
-        <Icon
-            color={props.isActive ? theme.TYPE_DARK_GREY : theme.TYPE_LIGHT_GREY}
-            size={24}
-            icon={props.icon}
-        />
-    ) : (
-        props.iconComponent
-    );
-
-    if (props.isMobileLayout) {
-        return (
-            <MobileWrapper {...props}>
-                <MobileIconWrapper isActive={props.isActive}>
-                    {iconComponent}
-                    {props.indicator === 'alert' && (
-                        <AlertDotWrapper>
-                            <AlertDot />
-                        </AlertDotWrapper>
-                    )}
-                    {props.indicator === 'check' && (
-                        <Indicator>
-                            <Icon icon="CHECK" size={10} color={theme.TYPE_GREEN} />
-                        </Indicator>
-                    )}
-                </MobileIconWrapper>
-                <Label>{props.label}</Label>
-            </MobileWrapper>
+export const ActionItem = React.forwardRef(
+    (props: ActionItemProps, ref: React.Ref<HTMLDivElement>) => {
+        const theme = useTheme();
+
+        const iconComponent = props.icon ? (
+            <Icon
+                color={props.isActive ? theme.TYPE_DARK_GREY : theme.TYPE_LIGHT_GREY}
+                size={24}
+                icon={props.icon}
+            />
+        ) : (
+            props.iconComponent
         );
-    }
 
-    return (
-        <HoverAnimation isHoverable={!props.isOpen}>
-            <Wrapper isActive={props.isActive} isOpen={props.isOpen} ref={ref} {...props}>
+        const Content = (
+            <>
                 {iconComponent}
                 {props.indicator === 'alert' && (
                     <AlertDotWrapper>
                         <AlertDot />
                     </AlertDotWrapper>
                 )}
+                {props.indicator === 'loading' && (
```

Comment:
Yellow spinner would be better?

Reply:
my personal feeling is that it's too small, have you checked that with ondra?

Reply:
I did not. @dahaca is our styling chief

Reply:
Yeah, I was considering other colors actually, the default one seemed neutral enough, will try yellow. I was also considering it's size, but it should be in like with the other indicators too. Ondra is away for now anyways but for now I think it's sufficient. Perhaps we will need another animated loader altogether.